guides/models/relationships.md

```diff
@@ -422,18 +410,18 @@ export default class AdeleRoute extends Route {
     return this.store
       .query('artist', {
         filter: { name: 'Adele' },
-        include: 'albums'
+        include: 'albums',
       })
-      .then(function(artists) {
+      .then(function (artists) {
         return artists[0];
       });
   }
 }
 ```
 
-### Updating Existing Records
+### Mise à jour des records existants
```

Comment:
To keep things consistent with the other titles, how they "speak to the reader".

```suggestion
### Mettre à jour des _records_ existants
```